queries/elixir.scm

```diff
@@ -0,0 +1,41 @@
+(comment) @comment
+(anonymous_function) @anonymousFunction
+(arguments) @arguments_list
+(call
+  target: (_) @functionCallee) @functionCall @functionCallee.domain
+
+;;(keywords
+;;  (pair
+;;    key: (_) @collectionKey @collectionItem.trailing.end.startOf
+;;    value: (_) @value) @collectionItem @collectionItem.trailing.start.endOf
+;;    (",")? @collectionItem.trailing.start.startOf
+;;    (#allow-multiple! @collectionItem))
+
+;; How does surrounding whitespace work for this
+(map_content
+  (binary_operator
+      left: (_) @collectionKey @collectionKey.trailing.start.endOf
+      right: (_) @value @collectionKey.trailing.end.startOf) @collectionItem
```

Comment:
you want the `binary_operator` to be the domain for key and value, and we'll want a test for that